Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -373,7 +373,7 @@ void BluetoothReal::WaitForHCICommandComplete(const u16 opcode)
   for (int tries = 0; tries < 100; ++tries)
   {
     if (libusb_interrupt_transfer(m_handle, HCI_EVENT, buffer.data(),
-                                  static_cast<int>(buffer.size()), &actual_length, 20) == 0 &&
+                                  static_cast<int>(buffer.size()), &actual_length, TIMEOUT) == 0 &&
```

Comment:
I have ~9ms of latency on my LAN (I use power-line communication between my steam link and my network switch, better than WiFi but not as great as Ethernet directly), so it's not a big latency. However, apparently the GC adapter is not working with my setup because I get a "latency burst" each time I launch a game and that makes the GC adapter thread stop because of an error.
The goal of my fix is mostly to try to avoid breaking an adapter connection in case a few requests gets delayed by a network latency burst.

Regarding WaitForHCICommandComplete, it seems like it is not called that often, and that it can already take up to 2 seconds to execute (100x 20ms timeout before returning).
Other timeouts are set to 200ms. Why not change this one too?
It will still take up to 2s to execute (10x 200ms), but it would absorb some more network latency, and it would even make calls take less time in case of a network shared bluetooth adapter (it would fail after 2 seconds with the current code for a 30ms latency for example, and it would succeed in 30ms if the timeout is bumped to 200ms).

Of course, my goal is not to be able to use the adapter over the internet for example. It's just to handle more "diverse" home networks that can have a latency in the tens of ms.